File: C#/IsPalindrome.cs
public class Solution {
    public bool IsPalindrome(int x) {
        
        if (x < 0)
        {
            return false;
        }
        
        int xOG = x;
        int xFlipped = 0;
        
        int length = x.ToString().Length;
        
        for (int i = 0; i < length; i++)
        {
            xFlipped *= 10;
            xFlipped += x % 10;

            x /= 10;
        }
        
        if (xFlipped == xOG)
        {
            return true;
        }
        else
        {
            return false;   
        }
    }
}

File: Python/isPalindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower()

        sOne = ""
        sTwo = ""
        for c in s:
            if c.isalnum():
                sOne = sOne + c
                sTwo = c + sTwo

        return (sOne == sTwo)